discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -246,5 +245,5 @@ def cid_is_valid(self, cid):
         try:
             make_cid(cid)
             return True
-        except:
+        except Exception as e:
```

Comment:
@raymondjacobson @jowlee i forsee potential conflicts with this pr https://github.com/AudiusProject/audius-protocol/pull/776

Reply:
nice, @piazzatron also fixed this in his PR. unrelated lint issue. will fix in rebase. thanks for the eye :)